src/Tools/GenerateAnalyzerNuspec/Program.cs

```diff
@@ -164,7 +164,7 @@
     foreach (string file in fileList)
     {
         var fileWithPath = Path.IsPathRooted(file) ? file : Path.Combine(projectDir, file);
-        result.AppendLine(FileElement(fileWithPath, "build"));
+        result.AppendLine(FileElement(fileWithPath, "buildTransitive"));
```

Comment:
I am afraid this might break our insertion pipeline of the analyzers into the .NET SDK. @jmarolf definitely needs to review this change.

Reply:
This seems to have broken mapping AnalysisLevel/AnalysisMode properties to the corresponding globalconfig files in the `build\config` folder. I am preparing a fix as part of another change.